doc/source/programs/gdal_vector_rasterize.rst

```diff
@@ -0,0 +1,214 @@
+    --target-resolution <xres> <yres>                        Set the target resolution [2 values]
+                                                            Mutually exclusive with --target-size
+    --tap                                                    (target aligned pixels) Align the coordinates of the extent of the output file to the values of the target-resolution
+    --target-size <xsize> <ysize>                            Set the target size in pixels and lines [2 values]
+                                                            Mutually exclusive with --target-resolution
+    --ot, --datatype, --output-data-type <OUTPUT-DATA-TYPE>  Output data type. OUTPUT-DATA-TYPE=Byte|Int8|UInt16|Int16|UInt32|Int32|UInt64|Int64|CInt16|CInt32|Float32|Float64|CFloat32|CFloat64
+    --optimization <OPTIMIZATION>                            Force the algorithm used (results are identical).. OPTIMIZATION=AUTO|RASTER|VECTOR (default: AUTO)
+
+    Advanced Options:
+    --oo, --open-option <KEY=VALUE>                          Open options [may be repeated]
+    --if, --input-format <INPUT-FORMAT>                      Input formats [may be repeated]
+
+    For more details, consult https://gdal.org/programs/gdal_vector_rasterize.html
+
+
+Description
+-----------
+
+:program:`gdal vector rasterize` burns vector geometries into a raster.
+
+The following options are available:
+
+Standard options
+++++++++++++++++
+
+.. include:: gdal_options/of_raster_create_copy.rst
+
+.. include:: gdal_options/co.rst
+
+.. option:: --band, -b <BAND>
+
+    The band(s) to burn values into. May be repeated.
+
+.. option:: --invert
+
+    Invert rasterization. Burn the fixed burn value, or the burn value associated with the first feature into all parts of the image not inside the provided polygon.
+
+    .. note::
+
+        When the vector features contain a polygon nested within another polygon (like an island in a lake), GDAL must be built against GEOS to get correct results.
+
+.. option:: --all-touched
+
+    Enables the ALL_TOUCHED rasterization option.
+
+    .. note::
+
+       When this option is enabled, the order of the input features (lines or polygons) can affect the results. When two features touch each other, the last one (i.e. topmost) will determine the burned pixel value at the edge. You may wish to use the `--sql` option to reorder the features (ORDER BY) to achieve a more predictable result.
+
+.. option:: --burn <BURN>
+
+    Burn value. May be repeated.
+
+.. option:: -a, --attribute-name <ATTRIBUTE-NAME>
+
+    Attribute name.
+
+.. option:: --3d
+
+    Indicates that a burn value should be extracted from the "Z" values of the feature. Works with points and lines (linear interpolation along each segment). For polygons, works properly only if the are flat (same Z value for all vertices).
+
+.. option:: --add
+
+   Instead of burning a new value, this adds the new value to the existing raster. Suitable for heatmaps for instance.
```

Comment:
Add some warning that the added values may overflow the limits of the datatype; 255+255 is too much for byte.